lmnet/lmnet/networks/object_detection/lm_fyolo.py

```diff
@@ -81,11 +94,14 @@ def base(self, images, is_training):
         darknet_block = partial(darknet, is_training=is_training,
                                 activation=self.activation, data_format=self.data_format)
 
-        x = darknet_block("block_1", self.inputs, filters=32, kernel_size=1)
+        x = self.encode_image(self.inputs)
```

Comment:
@yd8534976 
Blueoil has Quantize First Layer option in `blueoil.sh init` command.
Is it possible if Quantize First Layer is `true` then these code are enable,
If Quantize First Layer is `false` then these code are disable?